tests/Makefile

```diff
@@ -6,7 +6,11 @@ include ${PROJECT_REL_DIR}/common.mk
 LOCAL_POSTMAN_ENVIRONMENT=Docker.postman_environment.json
 
 MARTIN_ENV_OPTS=
-MARTIN_COMMON_OPTS=-v "$(CURDIR)/${PROJECT_REL_DIR}":/etc/postman -w /etc/postman/tests ${MARTIN_ENV_OPTS} --entrypoint sh
+MARTIN_BIND_SOURCE=${PROJECT_ABS_DIR}
+MARTIN_BIND_DEST=/etc/postman
+MARTIN_BIND=${MARTIN_BIND_SOURCE}:${MARTIN_BIND_DEST}
+
+MARTIN_COMMON_OPTS=-v ${MARTIN_BIND} -w /etc/postman/tests ${MARTIN_ENV_OPTS} --entrypoint sh
```

Comment:
Minor detail, but since you set the `MARTIN_BIND_DEST` var, might as well set the `-w` param to a subdir of that.